### ISSUE TITLE
Two versions of emacs on OSX?

### ISSUE THREAD

**Issue by Cumol** [jkitchin/scimax#26]
I seem to have two different versions of Emacs on my system. A 26 and 24.5 one. When I open scimax through the terminal, it opens up in the 24.5 version. Is it possible to switch to the newer one?

**Comment by jkitchin** [comment]
how do you open scimax? Usually I make a shell command like:

emacs -q -l `pwd`/scimax/init.el

so if you make two versions that point to your emacs-24.5 or 26 you could run either version.

**Comment by Cumol** [comment]
This is how I am opening scimax. If I use this in the Terminal it opens the 24.5 version.

`which emacs`

outputs:

`/usr/local/bin/emacs`

I tried to change emacs with an alias like this:

`alias emacs="/usr/local/Cellar/emacs/HEAD-38d0276/Emacs.app/Contents/MacOS/Emacs -nw"`

which should point then to Emacs 26.0 that I downloaded using Homebrew but it does not lead to any change...

Maybe I am missing something fundamental as I am rather new to Emacs/Terminal

**Comment by Cumol** [comment]
Managed to find a solution for my problem.

For whatever weird reason, the homebrew script in the wiki installed both emacs and emacs-plus and OSX kept taking emacs-plus as the standard emacs.

The moment I uninstalled emacs-plus with

`brew uninstall emacs-plus`

Terminal went back to the built-in emacs (version 22.1).

My solution now was to open the init.el file by the direct path to emacs like this:
`/Applications/Emacs.app/Contents/MacOS/Emacs "$@" -q -l scimax/init.el`

to make it easier I made an alias to this command and added it to my ~/.bashrc file
`alias scimax="/Applications/Emacs.app/Contents/MacOS/Emacs "$@" -q -l scimax/init.el"`

So invoking scimax in terminal now opens the new emacs (version 26) with the scimax init.el file
